src/main/resources/assets/onebar/lang/zh_cn.json

```
@@ -0,0 +1,131 @@
+{
+  "modmenu.descriptionTranslation.onebar": "只在你需要时显示长条和数值的界面模组",
+  "text.onebar.health": "命%s",
+  "text.onebar.absorption": "吸%s",
+  "text.onebar.hunger": "饿%s",
+  "text.onebar.resistance": "抗%s%%",
+  "text.onebar.invisibility": "隐",
+  "text.onebar.air": "气%s",
+  "text.onebar.freeze": "冷%s",
+  "text.onebar.fire": "火%s×",
+  "text.onebar.warden": "监%s",
+  "text.onebar.hardcore": "HC",
```

```suggestion
  "text.onebar.hardcore": "极限",
```

(From Minecraft Crowdin)